packages/smarthr-ui/src/components/ComboBox/MultiComboBox.tsx

```diff
@@ -527,7 +526,7 @@ const ActualMultiComboBox = <T,>(
             onCompositionStart={handleCompositionStartInput}
             onCompositionEnd={handleCompositionEndInput}
             onKeyDown={handleInputKeyDown}
-            autoComplete="off"
+            autoComplete={inputAttributes?.autoComplete || 'off'}
```

Comment:
```suggestion
            autoComplete={inputAttributes?.autoComplete ?? 'off'}
```

この書き方、 `0` や `false` を `"off"` に書き換えてしまうので `??` のほうがいいかもと思いました

合わせて、もしかしたら `"off"` はChromeで無視されるかも…？という気が付きがあったので共有しておきます 😭

Reply:
確かにー! 修正しました! https://github.com/kufu/smarthr-ui/pull/4664/commits/4fd54fad4350c03b1ea786cc8837be06b107637e
`"off"` は仕様どおりではあるので、現状のままで良いと判断します。 https://momdo.github.io/html/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute